pkgs/applications/window-managers/qtile/default.nix

```diff
@@ -33,13 +25,13 @@ let
 
     nativeBuildInputs = [
```

Comment:
It's possible to move both `pkg-config` and `setuptools-scm` under the same `with python3Packages`, feel free to not do that though since it's and confusing